src/Maestro/Maestro.Data/Models/Build.cs

```diff
@@ -22,24 +24,59 @@ public class Build
 
         static Build()
         {
+            IConfigurationRefresher configRefresher = null;
+            IConfiguration dynamicConfigs = null;
+            string appConfigurationConnString = Environment.GetEnvironmentVariable("AppConfigurationConnectionString");
```

Comment:
How is the connection string making it into the environment? Feels like this is something that should be getting pulled from a keyvault.

Reply:
Initial idea was to set this on the Azure Portal but after discussion with @alexperovich and @mmitche we decided to give Azure Managed Service Identity a shot. I updated the code using that.